apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.ts

```diff
@@ -52,22 +100,84 @@ router.post(
     if (!config.openAiApiKey || !config.openAiOrganization) {
       throw new error.HttpStatusError(403, 'Not implemented (feature not available)');
     }
+
     const client = new OpenAI({
       apiKey: config.openAiApiKey,
       organization: config.openAiOrganization,
     });
 
-    if (req.body.__action === 'generate_question') {
-      const jobSequenceId = await generateQuestion(
+    if (req.body.__action === 'ai_generate') {
+      const jobSeq = await generateQuestion(
```

Comment:
I'd call this something like `result`. This is not a job sequence.

Reply:
changed in e0331beaf9d587990dcda087d800feedc786fc87.